frontend/e2e/$lang+/_public/apply/adult-child/adult-child-flow.spec.ts

```diff
@@ -0,0 +1,415 @@
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to apply for your children page if applicant has no DTC, child is under 18', async () => {
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/apply-children/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Apply for your child(ren)' })).toBeVisible();
+    });
+
+    await test.step('Should navigate to dob eligibility page if applicant is 18-64, has no DTC, child is not under 18', async () => {
+      await page.getByRole('link', { name: 'Back' }).click();
+      await page.getByRole('link', { name: 'Back' }).click();
+
+      // Back to date of birth
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/date-of-birth/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Age' })).toBeVisible();
+      await fillOutDOB(page, '1985', '02', '10', 'No');
+
+      // Back to DTC
+      await page.getByRole('button', { name: 'Continue' }).click();
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/disability-tax-credit/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Disability tax credit' })).toBeVisible();
+      await page.getByRole('button', { name: 'Continue' }).click();
+
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/dob-eligibility/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Find out when you can apply' })).toBeVisible();
+    });
+
+    await test.step('Should navigate to apply for yourself page if applicant is 18-64, has DTC, child is not under 18', async () => {
+      await page.getByRole('link', { name: 'Back' }).click();
+      await page.getByRole('link', { name: 'Back' }).click();
+
+      // Back to date of birth
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/date-of-birth/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Age' })).toBeVisible();
+      await fillOutDOB(page, '1985', '02', '10', 'No');
+      await page.getByRole('button', { name: 'Continue' }).click();
+
+      // Continue to DTC
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/disability-tax-credit/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Disability tax credit' })).toBeVisible();
+      await page.getByRole('radio', { name: 'Yes' }).check();
+      await page.getByRole('button', { name: 'Continue' }).click();
+
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/apply-yourself/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Apply for yourself' })).toBeVisible();
+    });
+
+    await test.step('Should navigate to living independently page if applicant is 16 or 17, child is under 18', async () => {
+      await page.getByRole('link', { name: 'Back' }).click();
+
+      await fillOutDOB(page, '2007', '03', '20', 'Yes');
```

Comment:
This test will fail next year when a person born in 2007 will be 18. One way around this would be have a function that returns the the date of birth based on the user's age and the current year.

This same comment applies to other tests in this file that deals with a person's age. I also noticed this similar issue is present in `adult-flow.spec.ts` as well.

Reply:
Added a function that calculate the date based on the given age and current date